### ISSUE TITLE
Is this repo still alive?

### ISSUE THREAD

**Issue by KevinBeckers** [Supermarx/SuperMarx#1]

**Comment by Wassasin** [comment]
Yes, this is still actively being developed, but not necessarily as a complete open source project.

**Comment by KevinBeckers** [comment]
Oke, is it still available for the public or is only for commercial use? I like to incorporate this in a project.
